data/data/openstack/topology/sg-master.tf

```diff
@@ -78,7 +78,7 @@ resource "openstack_networking_secgroup_rule_v2" "master_ingress_https" {
   ethertype         = "IPv4"
   protocol          = "tcp"
   port_range_min    = 6443
-  port_range_max    = 6445
+  port_range_max    = 6443
   remote_ip_prefix  = "0.0.0.0/0"
   security_group_id = openstack_networking_secgroup_v2.master.id
 }
```

Comment:
Feels like we can remove the `master_ingress_heapster` and `master_ingress_heapster_from_worker` rules below.

https://github.com/kubernetes-retired/heapster

Reply:
In the same vein, what requires us to open port 80 with `master_ingress_http` ? We should consider removing it.

Reply:
Since github prevents me from adding my comments to the lines I want, I'm adding them all here...

Is `master_ingress_bootstrap_etcd` still current? AWS doesn't open tcp ports 12379-12380.

Reply:
Can we also use tf 0.12 syntax in these files?

"${openstack_networking_secgroup_v2.master.id}" -> openstack_networking_secgroup_v2.master.id